packages/cfd/src/Helpers/constants.ts

```diff
@@ -139,7 +139,25 @@ const getMT5WebTerminalLink = ({
     const server = is_demo ? 'Deriv-Demo' : server_name;
     const login = loginid ?? '';
 
-    return `https://metatraderweb.app/trade?servers=${server}&trade_server=${server}${login && `&login=${login}`}`;
+    const getBaseUrl = () => {
+        if (is_demo) return 'https://mt5-demo-web.deriv.com/terminal';
```

Comment:
should we add these link in variables like we did for DerivEz etc.
@hirad-deriv @shontzu-deriv

Reply:
I really wanted to refactor this also, either use variables for all links, or function calls for all links i.e.

```JavaScript
MT5_URL = getMT5WebTerminalLink()
DXTRADE_URL = getDxtradeLink()
DERIVEZ_URL = getDerivezLink()
```

then the function will handle if it's web (real/demo) or mobile (android, ios, huawei)

that would require quite a massive refactoring, I think we should discuss on it first?

Reply:
The better approach is to use variables, kindly help to make changes accordingly.